release_notes.md

```diff
@@ -18,6 +18,12 @@ This file tracks release notes for the loop client.
 
 #### Breaking Changes
 
+* Failing to load configuration file specified by `--configfile` for any reason if now hard error.
```

Comment:
nit: wrap at 80

Reply:
Hmm, previous lines weren't 80-wrapped so I thought the limit doesn't apply. Will fix.

Reply:
Done.